extensions/binding/org.eclipse.smarthome.binding.mqttbroker/.gitignore

```diff
@@ -0,0 +1 @@
+OSGI-INF/*.xml
```

Comment:
please create the folder `OSGI-INF` and place the `.gitignore` with content `*.xml` inside. right now the folder is marked as new for git after checkout and build.